src/components/templates/ExternalRoom/ExternalRoom.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from "react";
+import { useHistory } from "react-router-dom";
+
+import { AnyVenue } from "types/venues";
+import { WithId } from "utils/id";
+import { openUrl } from "utils/url";
+
+import "./ExternalRoom.scss";
+
+export interface ExternalRoomProps {
+  venue: WithId<AnyVenue>;
+}
+
+export const ExternalRoom: React.FC<ExternalRoomProps> = ({ venue }) => {
+  const history = useHistory();
+
+  useEffect(() => {
+    openUrl(venue.zoomUrl ?? "");
+  }, [venue]);
+
+  if (venue.zoomUrl) {
+    return (
+      <div className="ExternalRoom__message">
+        <h4>
+          Please keep this tab open while you are directed to external content.
+        </h4>
+        <h4>Return to this tab when you are ready to explore some more!</h4>
+        <a rel="noreferrer" href={venue.zoomUrl} target="_blank">
+          Open room manually
+        </a>
+      </div>
+    );
+  } else {
+    return (
+      <p>
```

Comment:
proper classname

Reply:
fixed in 733405113371f979b4941af7d375adc1f1e3678c